pages/Transfer.qml

```diff
@@ -534,82 +534,436 @@ Rectangle {
 
         GridLayout {
             visible: persistentSettings.transferShowAdvanced && appWindow.walletMode >= 2
-            columns: 6
-
-            StandardButton {
-                id: sweepUnmixableButton
-                text: qsTr("Sweep Unmixable") + translationManager.emptyString
-                enabled : pageRoot.enabled
-                small: true
-                onClicked: {
-                    console.log("Transfer: sweepUnmixableClicked")
-                    root.sweepUnmixableClicked()
+            columns: 2
+            Layout.fillWidth: true
+            Layout.leftMargin: 10
+            columnSpacing: 20
+            rowSpacing: 10
+
+            RowLayout {
```

Comment:
Move this to a separate component

Reply:
Move columnSpacing and rowSpacing to other component?

Reply:
Make a component where you can set title, helptext, button1 button2 button title / onclick

Goal is that it will look like this (pseudocode):
```
AdvancedOptionsBlock {
    title: Outputs
    button1text: "Export"
    button1enabled: true
    button2text: "Import"
    ...
}
AdvancedOptionsBlock { ... }
AdvancedOptionsBlock { ... }
AdvancedOptionsBlock { ... }
```